.github/workflows/ci.yml

```diff
@@ -8,14 +8,15 @@ jobs:
     permissions:
       id-token: write
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
-      - name: Install Rust Stable
+      - name: Install Rust 1.77.0
```

Comment:
Can we note that automation should update this on a new release so that readers aren't left to wonder.

Reply:
I don't think that we necessarily have to update the version with each new release, I think that it's fine to keep e.g. `1.77.0` until we indeed need to update something. We don't yet have automation for bumping this (there's no renovatebot on `sync-team`, right?), so that comment probably wouldn't be accurate (?).

Reply:
Ah I thought there was renovatebot. Nevermind then!